web/js/init.js

```diff
@@ -56,13 +56,15 @@ function generateCityQuery(city) {
 }
 
 function get_encoded_url_for_query(cityLookalikeMap) {
-  var cityName = document.getElementById("myInput").value;
+  var cityName = document.getElementById("myCityInput").value;
+  var otherSearch = document.getElementById("otherSearchInput").value;
   cityName = cityName.toLowerCase();
   var negativeQuery = ' -"wanted" -"not verified" -"unverified" -"needed" -"required" -"need" -"leads" ';
   var verifiedQuery = ' verified ';
   var needListQuery = generateOrQuery(arr);
+  var otherQuery = otherSearch ? ' ' + otherSearch + ' ' : '';
```

Comment:
Instead of creating the separate query, we should add the other resource in the `arr` which are a list of resource on which we need to perform the query operation. Please do local testing using the `rtpcr` as the otherSearch keyword and checking whether we are getting the results.

Reply:
Done, though we need to make a copy of the array rather than pushing to the existing arr.

Tested locally with rtpcr, combination of rtpcr and checkbox, and combination of 'home care' and checkbox.